### DISCUSSION TITLE
Is Tauri only compatible with Visual Studio?

### DISCUSSION THREAD

**Discussion by anugrah478** [tauri-apps/tauri#8443] [General]
![Screenshot (75)](https://github.com/tauri-apps/tauri/assets/154414495/bececd13-456a-4a72-8380-57bea298a00f)
I am trying to run a code from a Tauri app in Vscode, but it shows the image above and states that the code is only supported in Visual Studio and not in Vscode. Is this correct, or is there another way to run the code?

**Comment by FabianLars** [comment]
You can use vscode to work on your tauri app but you need a Visual Studio or VS Build Tools installation to be able to compile rust apps.
Rust basically just needs the linker and the windows sdk, the vs ide itself doesn't matter.
So once you installed it (for example as described in our [docs](https://tauri.app/v1/guides/getting-started/prerequisites#setting-up-windows)) you can use vscode just fine.
